packages/gitgraph-core/src/commit.ts

```diff
@@ -0,0 +1,174 @@
+import { CommitStyle } from "./template";
+import Branch from "./branch";
+
+export interface CommitOptions {
+  author: string;
+  subject: string;
+  style: CommitStyle;
+  body?: string;
+  notes?: string;
+  refs?: string[];
```

Comment:
I wonder if that wouldn't be better not to be optional. We could default it to an empty array. Having to deal with `undefined` value is a bit painful.

Same for `parents`.

What do you think?